src/plugins/intel_gpu/src/graph/broadcast.cpp

```diff
@@ -95,6 +95,77 @@ std::vector<layout> broadcast_inst::calc_output_layouts(broadcast_node const& /*
 
 template std::vector<layout> broadcast_inst::calc_output_layouts<ov::PartialShape>(broadcast_node const& node, const kernel_impl_params& impl_param);
 
+kernel_impl_params broadcast_inst::get_agnostic_updated_params(kernel_impl_params const& orig_impl_param) {
+    auto orig_input_layout = orig_impl_param.get_input_layout();
+    auto updated_param = orig_impl_param;
+    const auto& primitive = updated_param.typed_desc<broadcast>();
+
+    // Extend input dimensions with ones
+    auto i_layout = updated_param.input_layouts[0];
+    auto o_layout = updated_param.output_layouts[0];
+
+    if (i_layout.is_static() && o_layout.is_static()) {
```

Comment:
Is this agnostic_updated_params for shape agnostic kernel? 
Then, it is to be used at runtime right? 
Then, the runtime should be always static.

Reply:
Removed the static condition